docs/contributing/translations.md

```diff
@@ -0,0 +1,32 @@
+# Translations
+Wagtail uses Transifex to translate the content for the admin interface. Our goal is to ensure that Wagtail can be used by those who speak many different languages. Translation of admin content is a great way to contribute without needing to know how to write code.
+
```

Comment:
might be good to add a callout with a link to the `i18n` page - so people do not get confused.